package.json

```
@@ -56,7 +56,7 @@
     "test:deploy": "test/deploy/validate-package.sh",
     "test:ts": "tsc -p test --noEmit",
     "test:integration": "jest test/integration",
-    "test:unit": "jest --collectCoverage test/unit",
+    "test:unit": "jest --config=./jest.unit.config.js test/unit",
```

Nit: maybe
```suggestion
    "test:unit": "jest --config=./jest.coverage.config.js test/unit",
```
because it's not specific to unit in any way.